README.md

```diff
@@ -240,7 +240,11 @@ for `playcanvas.com/project/10/overview/test_proj` the id is 10.
 
 All listed key-value pairs are necessary. You can split them between
 `.pcconfig` (in your home directory), `pcconfig.json` (in your project target directory),
-and environment variables. `PLAYCANVAS_TARGET_DIR` cannot be set in `pcconfig.json`.
+and environment variables.
+
+`PLAYCANVAS_TARGET_DIR` can only be set in `.pcconfig` or an environment variable. You
+can also set `PLAYCANVAS_USE_CWD_AS_TARGET` to `1` in `.pcconfig` to use
```

Comment:
Why '1' and not true/false?

Reply:
Some json parsers may not support 'true', plus one can misspell it.